disnake/user.py

```diff
@@ -216,6 +224,16 @@ def avatar_decoration(self) -> Optional[Asset]:
             return None
         return Asset._from_avatar_decoration(self._state, self._avatar_decoration_data["asset"])
 
+    @property
+    def collectibles(self) -> Collectibles:
```

Comment:
This needs to be added to https://github.com/DisnakeDev/disnake/blob/8e1bd798e104d7b1a8eeb7a104c07423b7250372/disnake/member.py#L281 to prevent pyright from complaining about `<member>.collectibles`